### DISCUSSION TITLE
Problem with sorting entries in collection

### DISCUSSION THREAD

**Discussion by japgroevemaker** [statamic/cms#9025] [Troubleshooting]
Hello,

I have the same issue that is being discussed in this discussion: [#4481](https://github.com/statamic/cms/discussions/4481)
The name of my collection is Cases and this are the settings of the collection. 

![image](https://github.com/statamic/cms/assets/22168803/75173e32-1b29-4571-965b-51fed033c9a7)

I'm using Statamic with a database but the settings of the collection are still saved as `.yaml`

Can anyone help me out with this issue?

Thanks in advance

**Comment by duncanmcclean** [comment]
Does it happen on all collections or just one of them?
